Marlin/ultralcd.cpp

```diff
@@ -836,7 +836,9 @@ void kill_screen(const char* lcd_msg) {
 
     void _lcd_user_gcode(const char * const cmd) {
       enqueue_and_echo_commands_P(cmd);
-      lcd_completion_feedback();
+      #ifdef USER_SCRIPT_FEEDBACK
```

Comment:
OK, I _think_ I understand it now.

`#ifdef THING` and `#if defined(THNG)` are functionally the same if there is one condition, but `#if defined` allows multiple conditions.  I should use this when checking if `THING` is defined with a value.

`#if ENABLED(THING)` checks if `THING` is defined and doesn't expect it to have a value.

In this case I should use `#if ENABLED(USER_SCRIPT_FEEDBACK)` because I'm not assigning `USER_SCRIPT_FEEDBACK` a value, just defining it.  

(As opposed to, for example, `USER_SCRIPT_DONE`, which is assigned a value so the correct conditional statement is `#ifdef USER_SCRIPT_DONE`.)

PR updated.